tests/rt_intel_compile_only.conf

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Why do we need these two compile only rt.conf files?

Reply:
I accidentally committed them, let me remove them right away. Thanks for spotting this!